scripts/generator/graphql_generator/csharp/SDK/ShopifyError.cs.erb

```diff
@@ -16,7 +16,7 @@ namespace <%= namespace %>.SDK {
         }
 
         public enum ErrorType {
-            HTTP, GraphQL, UserError
+            HTTP, GraphQL, UserError, CheckoutError
```

Comment:
Perhaps: `NativePaymentProcessingError`, since this case happens when a native library fails